pkg/project/project.go

```diff
@@ -252,6 +262,46 @@ func (p *Project) Namespace() *Namespaced {
 // Environments returns project environment
 func (p *Project) Environments() string { return p.projectfile.Environments }
 
+func ResetProjectFile() {
+	projectfile.Reset()
+}
+
+func ParseVersionInfo(path string) (*projectfile.VersionInfo, error) {
+	return projectfile.ParseVersionInfo(path)
+}
+
+func GetProjectFilePath() (string, error) {
+	return projectfile.GetProjectFilePath()
+}
+
+func MakeEventsFromConstrainedEntities(items []projectfile.ConstrainedEntity) []*projectfile.Event {
+	return projectfile.MakeEventsFromConstrainedEntities(items)
+}
+
+func MakeScriptsFromConstrainedEntities(items []projectfile.ConstrainedEntity) []*projectfile.Script {
+	return projectfile.MakeScriptsFromConstrainedEntities(items)
+}
+
+type ConfigGetter interface {
+	projectfile.ConfigGetter
+}
+
+func GetProjectPaths(config ConfigGetter, namespace string) []string {
+	return projectfile.GetProjectPaths(config, namespace)
+}
+
+func GetProjectMapping(config ConfigGetter) map[string][]string {
+	return projectfile.GetProjectMapping(config)
+}
+
+func CleanProjectMapping(config ConfigGetter) {
+	projectfile.CleanProjectMapping(config)
+}
+
+func GetProjectNameForPath(config ConfigGetter, path string) string {
+	return projectfile.GetProjectNameForPath(config, path)
+}
```

Comment:
Sorry, I think this is just obfuscating the problem. In many cases the new equivalent of these would either be automated at a lower level or have a different API altogether. Please revert these.